packages/console/src/pages/SignInExperience/tabs/SignUpAndSignInTab/components/ConnectorSetupWarning.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+  ConnectorResponse,
+  ConnectorType,
+  SignInIdentifier,
+  SignUpIdentifier,
+} from '@logto/schemas';
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { snakeCase } from 'snake-case';
+import useSWR from 'swr';
+
+import Alert from '@/components/Alert';
+import { RequestError } from '@/hooks/use-api';
+
+import { signUpToSignInIdentifierMapping } from '../constants';
+
+type Props = {
+  signUpIdentifier?: SignUpIdentifier;
+  signInIdentifiers?: SignInIdentifier[];
+};
+
+const getNoConnectorI18nContext = (connectorTypes: ConnectorType[]) => {
+  const { length } = connectorTypes;
+
+  if (length === 1) {
+    return connectorTypes[0]?.toLocaleLowerCase() ?? '';
+  }
+
+  if (
+    length === 2 &&
+    connectorTypes.includes(ConnectorType.Email) &&
+    connectorTypes.includes(ConnectorType.Sms)
+  ) {
+    return [ConnectorType.Email, ConnectorType.Sms].join('_or_').toLocaleLowerCase();
+  }
+
+  return '';
+};
```

Comment:
https://www.i18next.com/translation-function/formatting#list i18n could auto format list

Reply:
Removed for we have a new design requirement.